solidity/contracts/TransformerOracle.sol

```diff
@@ -218,15 +236,7 @@ contract TransformerOracle is BaseOracle, AccessControl, ITransformerOracle {
       );
   }
 
-  function _getTransformers(address _tokenA, address _tokenB)
-    internal
-    view
-    returns (ITransformer _transformerTokenA, ITransformer _transformerTokenB)
-  {
-    return _getTransformersOptimized(_tokenA, _tokenB, true, true);
-  }
-
-  function _getTransformersOptimized(
+  function _getTransformers(
```

Comment:
Deleted `_getTransformers` and renamed `_getTransformersOptimized` to `_getTransformers`